### DISCUSSION TITLE
ERS and RADARSAT from ASF

### DISCUSSION THREAD

**Discussion by mfangaritav** [isce-framework/isce2#678] [Q&A] [answered]
Hello

I was wondering if someone has tried to process ERS or RADARSAT data from ASF using ISCE. I tried to process some interferograms with RADARSAT images level 0 but the .raw and .ldr files from ASF do not work on ISCE. I found a note in the template file for RADARSAT

"ISCE can currently only handle RADARSAT-1 in CEOS format available from UNAVCO / WInSAR. We do not support ASF's STF format yet."

I was wondering if there's a way to convert the files from ASF so they can be processed with ISCE.

Thanks

**Comment by EJFielding** [comment, marked as answer]
I don't know of any Open Source software that can process the RADARSAT-1 STF (Sky Telemetry Format) files into the standard CEOS frames that ISCE2 supports. The commercial GAMMA software used to support RADARSAT-1 STF processing, and it may still support it.

**Reply by mfangaritav** [reply]
Thanks Eric, I'll download the files from ESA directly instead.
